app/redux/reducers/connectionReducer.js

```diff
@@ -0,0 +1,11 @@
+let connectionReducer = function(connectionStatus='Connect To Server', action) {
```

Comment:
I was trying to split the reducer and combine at at an index for js. Was facing an error so let that be. (Noted @iffyuva 's comment of not checking in files that aren't used.)